sandbox-backend/src/main/resources/db/migration/V0000001__addTag.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO tag (tagName) VALUES ('sport');
```

Comment:
We already have V000001 migration, please rename file into V{last_migration + 1}